independent-publisher-connectors/OpenAI/apiDefinition.swagger.json

```diff
@@ -87,20 +87,25 @@
             "name": "engine",
             "in": "path",
             "type": "string",
-            "description": "The used engine, choose between text-davinci-002, text-curie-001, text-babbage-001, text-ada-001",
+            "description": "The used engine, choose between text-davinci-002/003, text-curie-001, text-babbage-001, text-ada-001",
```

Comment:
@PowerRobin text-davinci-003 is essentially Chat GPT, is that correct? If yes, can you please include this in the description?

Reply:
I got a feedback the text-danvinci-003 isn't entirely chat GPT but close. Let's keep it as is an publish. @Amjed-Ayoub

Reply:
ChatGPT is derived from GPT3 text-danvinci-003 model. So basically siblings, but optimized for different things.
Would also like to keep it as is.